Tests/XCUITests/DesktopModeTests.swift

```diff
@@ -23,10 +23,12 @@ class DesktopModeTestsIpad: IpadOnlyTestCase {
 
         // Covering scenario that when reloading the page should preserve Desktop site
         navigator.performAction(Action.ReloadURL)
+        waitUntilPageLoad()
         XCTAssert(app.webViews.staticTexts.matching(identifier: "MOBILE_UA").count > 0)
 
-        navigator.goto(TabTray)
-        navigator.goto(CloseTabMenu)
```

Comment:
@dragosb01 I've been investigating the failure on `testLongPressReload()` on the side, too. I still get the test to fail intermittently on my end by going to the tab tray right away. 😞

Reply:
I use the option `-test-iterations 100` on the command line to test the change.

Reply:
even after rebasing main with his PR?

Reply:
Yes, the test fails on my end even after pulling the changes from `main`. 😞 I don't like this workaround, but we may use this (draft) PR to talk to the developers.

Reply:
Oh no. Bitrise does not like the change. 😞 
https://addons-testing.bitrise.io/builds/7d07efd1-c63d-45ed-9384-7872a8dee33e/testreport/99e6482e-a0f7-4ac2-ab9c-201c280c1687/testsuite/0/testcases?status=failed

Reply:
😄 I need to ensure that the screen graph to know which screen we're in. The change should be ok.
https://app.bitrise.io/app/6c06d3a40422d10f/pipelines/03240ef1-cffc-4dc8-a643-7fcd35dcfe07